client/less/code-mirror.less

```diff
@@ -34,4 +34,9 @@
 
 .CodeMirror-lint-tooltip {
   z-index: 9999;
-}
\ No newline at end of file
+}
+
+div.CodeMirror span.CodeMirror-matchingbracket {
+  border: 0.5px solid #939393 !important;
+  color: #000;
```

Comment:
This `color` doesn't work in night mode, the bracket completely disappears.

Perhaps we could make it `#939393` also, so the user knows what type of bracket is highlighted.

Reply:
@Bouncey I think `#939393` should be set as the color for brackets only in night mode and `#000000` in the normal state. 
What do you think?